app/services/articles/unpublish.rb

```diff
@@ -0,0 +1,15 @@
+module Articles
+  class Unpublish
```

Comment:
I would use `module` + `module_function` because we don't instantiate the class, but that's the personal preference :eyes: